index.md

```diff
@@ -48,6 +48,16 @@ links:
 * [whitepaper](https://github.com/cloudspannerecosystem/harbourbridge/blob/master/whitepaper.md)
 * [open issues](https://github.com/cloudspannerecosystem/harbourbridge/issues)
 
+### spanner-cli
+
+The spanner-cli is an interactive command line tool for Cloud Spanner.
```

Comment:
Suggesting some very minor changes:

spanner-cli is an interactive command line tool, inspired by the `mysql` utility. It allows you to control your Cloud Spanner databases with idiomatic database commands, such as `SHOW TABLES` or
SELECT/INSERT/UPDATE/DELETE`.

Reply:
Thanks for your suggestion. It sounds better!